scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,120 @@
+@mixin vf-p-navigation-reduced {
+  @media (min-width: $breakpoint-navigation-threshold - 1) {
+    .p-navigation--sliding.is-reduced.has-search-open {
+      .p-navigation__nav .p-navigation__items {
+        display: inline-flex;
+      }
+
+      .p-navigation__search {
+        align-items: center;
+        background-color: #262626;
+        display: flex;
+        justify-content: center;
+        left: 0;
+        padding: 0.625rem 1rem;
+        position: absolute;
+        top: 2rem;
+        width: 100%;
+
+        .p-search-box {
+          margin: 0;
+          max-width: 50rem;
+        }
+      }
+
+      .p-navigation.is-secondary {
+        display: none;
+      }
+    }
+  }
+
+  .p-navigation--sliding.is-reduced {
```

Comment:
Is there a reason for this being defined after `has-search-open`? It's a bit confusing to have specific search styles first, followed by more generic ones.

Reply:
I have moved some stuff around which hopefully addresses this